Gen2-Sample/CSharp-TSI-Gen2-Sample/DataPlaneClient/azure-rest-api-specs/specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/examples/WarmStoreEventSchemaGet.json

```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {
-    "api-version": "2018-11-01-preview",
-    "storeType":  "warmstore",
+    "api-version": "2020-07-31",
+    "storeType": "warmstore",
     "environmentFqdn": "10000000-0000-0000-0000-100000000109.env.timeseries.azure.com",
     "parameters": {
       "searchSpan": {
```

Comment:
same as above: Add event schema example with Long and Dynamic data types, with escaping, etc. shown?